narwhal/worker/src/batch_maker.rs

```diff
@@ -34,43 +46,47 @@ pub struct BatchMaker {
     /// Receive reconfiguration updates.
     rx_reconfigure: watch::Receiver<ReconfigureNotification>,
     /// Channel to receive transactions from the network.
-    rx_batch_maker: Receiver<Transaction>,
+    rx_transaction: Receiver<(Transaction, TxResponse)>,
```

Comment:
I renamed `rx_transaction` to `rx_batch_maker`, so the receiver name matches the channel metric and is consistent across the codebase. I can rename this back in a followup PR, or is the name `rx_transaction` preferable?